src/types.ts

```diff
@@ -8,21 +8,26 @@ export type Suite = {
   }
 }
 
-export interface IHasPath {
-  path: string;
-}
-
 export type NpmConfig = {
   registry?: string;
   strictSSL?: boolean | string | null;
   packageLock?: boolean | string | null;
   packages?: { [key: string]: string | number };
 }
 
-export interface IHasNpmConfig {
+export interface PathContainer {
+  path: string;
+}
+
+export interface NpmConfigContainer {
   npm?: NpmConfig;
 }
 
-export interface IHasSuites {
+export interface SuitesContainer {
   suites: Suite[];
 }
+
+export interface NodePath {
```

Comment:
How about?

```suggestion
export interface NodeContext {
```

It's just that it might be confusing further down the call chain, when dealing with the `nodePath` string vs `NodePath` the interface.

Reply:
That makes totally sense !